editor.planx.uk/src/routes/index.tsx

```diff
@@ -30,6 +31,13 @@ const editorRoutes = mount({
         }),
   ),
 
+  "/:team/:flow/:sessionId/download-application": map((req) => {
+    return route({
+      title: makeTitle("Download application"),
+      view: <VerifySubmissionEmail params={req.params} />,
```

Comment:
As a user I'd expect this to still be within an existing PlanX view (e.g. with header and footer).

There's a good example of this on the invite to pay pages - I think it's just a cast of wrapping this in the `standaloneView()` function.

Reply:
Thanks for point that my way - looks loads better now! 🎉